tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/HugeDataTest.scala

```diff
@@ -38,6 +38,7 @@ class HugeDataTest extends AnyFunSuite
 
   test("celeborn spark integration test - huge data") {
     val sparkConf = new SparkConf().setAppName("rss-demo").setMaster("local[4]")
+      .set("spark.celeborn.shuffle.register.maxRetries", "10")
```

Comment:
just curious, what's the purpose to set this config?

Reply:
this ut will failed, because of `registerShuffleInternal` will return null. So I increase this conf.

Reply:
> 

Is this related to this pr?

Reply:
I can't reproduce, we should better to revert